client/utils/auth.py

```diff
@@ -19,12 +19,26 @@
 
 log = logging.getLogger(__name__)
 
+
 CLIENT_ID = \
     '931757735585-vb3p8g53a442iktc4nkv5q8cbjrtuonv.apps.googleusercontent.com'
 # The client secret in an installed application isn't a secret.
 # See: https://developers.google.com/accounts/docs/OAuth2InstalledApp
 CLIENT_SECRET = 'zGY9okExIBnompFTWcBmOZo4'
-REFRESH_FILE = '.ok_refresh'
+
+
+def get_config_directory():
+    if sys.platform == 'win32':
```

Comment:
What happens when someone is running on XP? Would we end up creating the AppData folder?

What if we keep the storage folder as always `~/.config/ok`? I know it's not "supposed" to be where you put files on windows - but it will work across platforms. 

If we want to be fully platform complaint with where we store files - we'd want to use AppDirs (we'd have to bundle this file into the ok source) 
https://github.com/ActiveState/appdirs/blob/master/appdirs.py